backend/init_data/skills/dingtalk-knowledge/provider.py

```
@@ -0,0 +1,768 @@
+            JSON response string
+        """
+        # Validate file size before upload
+        if file_size > ATTACHMENT_MAX_BYTES:
+            # Cleanup temp directory
+            try:
+                await sandbox.commands.run(
+                    cmd=["rm", "-rf", temp_dir],
+                    cwd="/home/user",
+                )
+            except Exception:
+                pass
```

Ruff (0.15.10)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)